datadog/resource_datadog_dashboard.go

```diff
@@ -8058,6 +8214,49 @@ func buildDatadogSplitGraphDefinition(terraformDefinition map[string]interface{}
 	return datadogDefinition, nil
 }
 
+func buildDatadogPowerpackDefinition(terraformDefinition map[string]interface{}) (*datadogV1.PowerpackWidgetDefinition, error) {
+	datadogDefinition := datadogV1.NewPowerpackWidgetDefinitionWithDefaults()
+	// Required params
+	//type, powerpack_id
+
+	powerpack_type, _ := datadogV1.NewPowerpackWidgetDefinitionTypeFromValue("powerpack")
+	datadogDefinition.SetType(*powerpack_type)
```

Comment:
Note: You might notice that "type" is a required property for powerpack widgets [[src](https://github.com/DataDog/datadog-api-spec/blob/master/spec/v1/dashboard.yaml#L1528)]. Since the only acceptable value is "powerpacks", I am setting it here instead of requiring users to add it in the TF definition.